integrations/slack/src/index.ts

```diff
@@ -27,6 +27,10 @@ const handleSpaceContentUpdated: EventCallback<
     }
 
     const { data: space } = await api.spaces.getSpaceById(event.spaceId);
+    const { data: revision } = await api.revisions.getRevisionById(event.spaceId, event.revisionId);
+
+    const revisionMessage = revision.git.message;
+    const pagesUpdatedInRevision = revision.pages.map((page) => page.title);
```

Comment:
Hello 👋, thanks for the contribution. It's really a neat idea. Though the solution will not work as expected because a `revision` does not contain only "updated pages" but instead it has reference to `all` the pages in the space. 

So, this is basically going to print `titles` of all the pages in the space which isn't what we want at the end of the day ;) 

On a related note, you can test integrations locally without having to publish again & again. Refer https://developer.gitbook.com/getting-started/development#local-development

Reply:
Thanks for linking me to the docs!

Is there a way to get only the pages that have been updated for that revision? If not, the Git message is probably good enough to bring a bit more context 😄

Reply:
Unfortunately no, not from the revisions. The Git message is also optional and present for only those spaces that has Git Sync set up 😄